pkg/configs/api/api_test.go

```diff
@@ -244,7 +245,20 @@ var amCfgValidationTests = []struct {
           email_configs:
           - to: myteam@foobar.org`,
 		shouldFail:  true,
-		errContains: "email notifications are not supported in Cortex yet",
+		errContains: "email notifications are disabled",
```

Comment:
Would it be possible to add a test covering the case email notifications are enabled?